packages/css-framework/src/components/_alert.scss

```diff
@@ -1,100 +1,45 @@
-$alert-states: (
-  neutral: (
-    background: color("neutral", 000),
-    border: color("neutral", 100),
-    copy: color("neutral", 600),
-    title: color("neutral", 900),
-  ),
-  primary: (
-    background: color("primary", 000),
-    border: color("primary", 100),
-    copy: color("primary", 800),
-    title: color("primary", 900),
-  ),
-  danger: (
-    background: color("danger", 000),
-    border: color("danger", 100),
-    copy: color("danger", 800),
-    title: color("danger", 900),
-  ),
-  warning: (
-    background: color("warning", 100),
-    border: color("warning", 300),
-    copy: color("warning", 800),
-    title: color("warning", 900),
-  ),
-  success: (
-    background: color("success", 000),
-    border: color("success", 200),
-    copy: color("success", 800),
-    title: color("success", 900),
-  ),
-);
-
 .rn-alert {
```

Comment:
In the 'without title' screenshot the line height looks a little bit off - maybe this could be vertically centered with `inline-flex` box and `align-items: center;`?